open_samus_returns_rando/files/templates/custom_init.lua

```diff
@@ -1,8 +1,10 @@
 Game.ImportLibrary("system/scripts/init_original.lua")
 
-Init.tNewGameInventory = {
-    TEMPLATE("new_game_inventory")
-}
+Init.tNewGameInventory = TEMPLATE("new_game_inventory")
```

Comment:
Does this work without the `{` `}` around it? I think that will lead to a wrong init file or I'm missing something.

Reply:
I tested after changing this and everything behaves like it did before